Tests/Cells/AsynchronousResizableBrick.swift

```diff
@@ -30,16 +29,23 @@ class AsynchronousResizableBrickCell: BrickCell, Bricklike, AsynchronousResizabl
 
     func fireTimer() {
         self.heightConstraint.constant = brick.newHeight
-        sizeChangedHandler?(self)
-        brick.didChangeSizeCallBack?()
+        self.resizeDelegate?.performResize(cell: self, completion: { [weak self] (completed: Bool) in
+            self?.brick.didChangeSizeCallBack?()
```

Comment:
I meant like `self.resizeDelegate?.performResize(cell: self, completion: { [weak self] _ in`